### DISCUSSION TITLE
instruct Ceph to downscale PG in parallel

### DISCUSSION THREAD

**Discussion by ghost** [rook/rook#10869] [Q&A]
Is it possible to instruct Ceph downscale PGs in parallel?
I've had 1024 PGs and the pg autoscaler have changed them down to 256 since I've removed some OSD's.

Watching `ceph status` reports pgs are getting down by 1 o 2 at a time..
Is there a way to increase the amount of PGs being shuffled? I'd like to double it.

**Comment by travisn** [comment]
@andy108369 Are you on Quincy? I don't see a specific setting for tuning the PG shuffling in parallel, but perhaps [this blog](https://ceph.io/en/news/blog/2022/autoscaler_tuning/) would have some ideas for you.

**Reply by ghost** [reply]
Yes, I am on Quincy. I've been through that blog, haven't found anything related to configuring the parallelism there... :man_shrugging:
